source/features/split-issue-pr-search-results.tsx

```diff
@@ -54,6 +54,8 @@ function init() {
 
 		const title = select('.codesearch-results h3').firstChild;
 		title.textContent = title.textContent.replace('issue', 'pull request');
+	} else if (/\btype=Issues\b/.test(location.search) && !/\bis:issue\b/.test(getSearchQuery())) {
+		issueLink.click();
```

Comment:
We can't do this, it will cause a refresh.

It's best to update the links on the page:

https://github.com/sindresorhus/refined-github/blob/329c2f25e9d85a62b0bc450d8e21507ab1d1a3e1/source/features/sort-milestones-by-closest-due-date.tsx

Reply:
[**@bfred-it**](https://github.com/bfred-it) commented on [Jan 31, 2019, 6:14 AM GMT+1](https://github.com/sindresorhus/refined-github/pull/1738#discussion_r252537327 "2019-01-31T05:14:23Z - Replied by Github Reply Comments"):
> But first, how do you get to this point?
> 
> /\\btype=Issues\\b/.test(location.search) && !/\\bis:issue\\b/.test(getSearchQuery()))

Steps to reproduce:
1. Go to https://github.com/search?q=typescript&type=Issues
2. ![image](https://user-images.githubusercontent.com/55841/52175393-6fc75e00-27a3-11e9-80fe-177ef0184905.png)

Any suggestions to correct the number in the results?

Reply:
Yes but who do you get to that URL? Can you only reach it by linking to it directly?

Reply:
In that case _perhaps_ we shouldn't alter the results, because someone linked to both PRs and Issues at the same time.

To do this:

- de-select the Issues nav
- add ` and pull requests` to the title

Reply:
The same can be reproduced by:
1. Go to https://github.com
2. Do a global search for `script`
3. Select "Issues" type
4. Alter the search value to `typescript`
5. Same results... ![image](https://user-images.githubusercontent.com/55841/52175393-6fc75e00-27a3-11e9-80fe-177ef0184905.png)

Reply:
Uhm, that doesn't happen because we already fix the "Issues" link on that page

Reply:
Indeed. We should capture `submit` and alter the `q` to match the current page (issues or PR)

Reply:
Sounds like the logical choice, but we're actually not in either link: clicking either one of those will show different content.

This (https://github.com/sindresorhus/refined-github/pull/1738#discussion_r253299205) is not too difficult to do and works for both the cases we talked about. The `submit` capture would be a plus, but that situation is also covered by (https://github.com/sindresorhus/refined-github/pull/1738#discussion_r253299205)

Reply:
Ehhhh I just saw your screenshot. Maybe it doesn't look too bad. Sometimes the "current page" highlighted in a menu doesn't actually link to itself but to the "top" view. So I'd be ok with your solution.

Reply:
[**@bfred-it**](https://github.com/bfred-it) commented on [Feb 3, 2019, 1:36 PM GMT+1](https://github.com/sindresorhus/refined-github/pull/1738#discussion_r253300337 "2019-02-03T12:36:08Z - Replied by Github Reply Comments"):
> Ehhhh I just saw your screenshot. Maybe it doesn't look too bad.
> 
> Sometimes the "current page" highlighted in a menu doesn't actually link to itself but to the "top" view. So I'd be ok with your solution.

I kinda like it too. It gives the user feedback that both results are shown (issues & PRs). But it also keeps the original urls, linking to their corresponding results (issues or PRs).

Reply:
Committed the change for selecting both menu items when the user has not decided which type should be used (issues or PRs), without reloading the page.